src/test/pythonnode_test.cpp

```diff
@@ -1003,7 +1003,7 @@ class MockedMediapipeGraphExecutorPy : public ovms::MediapipeGraphExecutor {
         const PythonNodeResourcesMap& pythonNodeResourcesMap,
         PythonBackend* pythonBackend,
         MediapipeServableMetricReporter* mediapipeServableMetricReporter) :
-        MediapipeGraphExecutor(name, version, config, inputTypes, outputTypes, inputNames, outputNames, pythonNodeResourcesMap, {}, {}, pythonBackend, mediapipeServableMetricReporter) {}
+        MediapipeGraphExecutor(name, version, config, inputTypes, outputTypes, inputNames, outputNames, pythonNodeResourcesMap, {}, {}, {}, pythonBackend, mediapipeServableMetricReporter) {}
```

Comment:
when are we switching unit tests to use single constructor, which is used in the actual product, instead of testing different constructor which exists for the purpose of unit tests now?

Reply:
![image](https://github.com/user-attachments/assets/00d49643-f85f-43ee-9f80-d0959bfca986)
So we shouldn't have those {}, {}. {}. {} and expand them each time for new servableTypeMap